db-scripts/src/main/resources/migration.sql

```diff
@@ -95,18 +95,18 @@ CREATE TABLE `clinical_attribute_meta` (
   PRIMARY KEY (`ATTR_ID`, `CANCER_STUDY_ID`),
   FOREIGN KEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
 );
-INSERT INTO clinical_attribute_meta 
-  SELECT DISTINCT clinical_sample.attr_id, clinical_attribute.display_name, clinical_attribute.description, clinical_attribute.datatype, clinical_attribute.patient_attribute, clinical_attribute.priority, cancer_study.cancer_study_id 
-  FROM clinical_attribute 
-  INNER JOIN clinical_sample ON clinical_attribute.ATTR_ID = clinical_sample.ATTR_ID 
-  INNER JOIN sample ON clinical_sample.internal_id = sample.internal_id 
-  INNER JOIN patient ON sample.patient_id = patient.internal_id 
+INSERT INTO clinical_attribute_meta
+  SELECT DISTINCT clinical_sample.attr_id, clinical_attribute.display_name, clinical_attribute.description, clinical_attribute.datatype, clinical_attribute.patient_attribute, clinical_attribute.priority, cancer_study.cancer_study_id
+  FROM clinical_attribute
+  INNER JOIN clinical_sample ON clinical_attribute.ATTR_ID = clinical_sample.ATTR_ID
+  INNER JOIN sample ON clinical_sample.internal_id = sample.internal_id
+  INNER JOIN patient ON sample.patient_id = patient.internal_id
   INNER  JOIN cancer_study ON patient.cancer_study_id = cancer_study.cancer_study_id;
-INSERT INTO clinical_attribute_meta 
-  SELECT DISTINCT clinical_patient.attr_id, clinical_attribute.display_name, clinical_attribute.description, clinical_attribute.datatype, clinical_attribute.patient_attribute, clinical_attribute.priority, cancer_study.cancer_study_id 
-  FROM clinical_attribute 
-  INNER JOIN clinical_patient ON clinical_attribute.ATTR_ID = clinical_patient.ATTR_ID 
-  INNER JOIN patient ON clinical_patient.internal_id = patient.internal_id 
+INSERT INTO clinical_attribute_meta
+  SELECT DISTINCT clinical_patient.attr_id, clinical_attribute.display_name, clinical_attribute.description, clinical_attribute.datatype, clinical_attribute.patient_attribute, clinical_attribute.priority, cancer_study.cancer_study_id
+  FROM clinical_attribute
+  INNER JOIN clinical_patient ON clinical_attribute.ATTR_ID = clinical_patient.ATTR_ID
```

Comment:
??? why is this part of migration.sql changing?

Reply:
Accidentally changed styling, I reverted it.